source/Octopus.Tentacle/Communications/HalibutInitializer.cs

```diff
@@ -101,6 +101,7 @@ void AddPollingEndpoints()
                 var halibutTimeoutsAndLimits = new HalibutTimeoutsAndLimits();
                 var serviceEndPoint = new ServiceEndPoint(pollingEndPoint.Address, pollingEndPoint.Thumbprint, halibutProxy, halibutTimeoutsAndLimits);
                 halibut.Poll(new Uri(pollingEndPoint.SubscriptionId), serviceEndPoint, CancellationToken.None);
+                halibut.Poll(new Uri(pollingEndPoint.SubscriptionId), serviceEndPoint, CancellationToken.None);
```

Comment:
@nathanwoctopusdeploy is it as simple as this?

Reply:
I think so 👍